src/digisos/index.tsx

```diff
@@ -1,31 +1,70 @@
-import * as React from "react";
-import {Redirect, Route, Switch} from "react-router";
-import SideIkkeFunnet from "../nav-soknad/containers/SideIkkeFunnet";
+import {createRoutesFromChildren, matchRoutes, Route, useLocation, useNavigationType} from "react-router";
+import SideIkkeFunnet from "../nav-soknad/feilsider/SideIkkeFunnet";
 import Informasjon from "./hovedmeny";
-import SkjemaRouter from "./skjema";
+import {erDev, erMockAlt} from "../nav-soknad/utils";
+import {createBrowserRouter, createRoutesFromElements, redirect} from "react-router-dom";
+import {basePath} from "../configuration";
+import * as React from "react";
 import Ettersendelse from "./skjema/ettersendelse";
-import Link from "./link";
-import {erMockMiljoEllerDev} from "../nav-soknad/utils";
+import Steg1 from "./skjema/personopplysninger";
+import Steg2 from "./skjema/begrunnelse";
+import Steg3 from "./skjema/arbeidUtdanning";
+import Steg4 from "./skjema/familie";
+import Steg5 from "./skjema/bosituasjon";
+import Steg6 from "./skjema/inntektFormue";
+import Steg7 from "./skjema/utgifterGjeld";
+import Steg8 from "./skjema/okonomiskeOpplysninger";
+import NyOppsummering from "./skjema/ny-oppsummering/Oppsummering";
 import * as Sentry from "@sentry/react";
+import {BrowserTracing} from "@sentry/tracing";
 
-const SentryRoute = Sentry.withSentryRouting(Route);
+const redirectFromLogin = async () => {
+    const url = window.location.href;
+    const match = url.match(/goto=\/sosialhjelp\/soknad(.+?)(&login_id.*$|$)/);
+    const destination = match?.[1] ? match[1] : "/informasjon";
+    return redirect(destination);
+};
```

Comment:
Jeg slet litt med dette i innsyn, for å få det rett både i mock-miljø og q0. Jeg har ikke sjekket q0 for søknaden, men innnloggingen funker bra der? Isåfall så dette litt smoothere ut enn sånn jeg gjorde det i innsyn :)